docs/form.mdx

```diff
@@ -136,21 +136,21 @@ A unique identifier for this plugin (`string`, default: `'Form'`).
 #### `target`
 
 DOM element or CSS selector for the form element (`string` or `Element`,
-default: `null`)
+default: `null`).
 
 This is required for the plugin to work.
 
 #### `resultName`
 
 The `name` attribute for the `<input type="hidden">` where the result will be
-added (`string`, default: `uppyResult`)
+added (`string`, default: `uppyResult`).
 
 #### `getMetaFromForm`
 
 Configures whether to extract metadata from the form (`boolean`, default:
-`true`)
+`true`).
 
-When set to true, the `Form` plugin will extract all fields from a `<form>`
+When set to `true`, the Form plugin will extract all fields from a `<form>`
```

Comment:
```suggestion
When set to `true`, the `Form` plugin will extract all fields from a `<form>`
```

Reply:
Do we need to mark the name of a plugin as code? Dashboard, Form, DragDrop.

Reply:
I use backticks to wrap any word that should not be "parsed" by the reader as an English word, but instead as some jargon. I don't have strong feelings here, maybe the fact that `Form` is capitalised is enough here.